reinforcement_learning/rl_cartpole_batch_coach/rl_cartpole_batch_coach.ipynb

```diff
@@ -0,0 +1,726 @@
+{
```

Comment:
<blockquote>As we can see in the plot, these metrics are getting improvements as the learning agent is iterating multiple epochs over the given dataset.</blockquote><p>As we can see in the plot, these metrics are <strong>improving</strong> as the learning agent is <strong>iterating</strong> over the given dataset.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/926/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>